### ISSUE TITLE
Cannot change networks while connected to one

### ISSUE THREAD

**Issue by krisgesling** [MycroftAI/mycroft-gui-mark-2#19]
**Describe the bug**
Selecting a new wifi network to connect to does not prompt for the network password. It jumps straight to the "Connecting to Wi-Fi" animation and gets stuck.

**To Reproduce**
1. Boot a Mark II and connect to the internet
2. From the pull down menu select "wireless settings"
3. Hit refresh to display all available networks
4. Select a network that you are not connected to
5. Jumps to "Connecting to Wi-Fi" animation
6. Reboot required to escape the animation.

**Expected behavior**
- When displaying wifi settings, available networks should populate automatically
- Whether or not an existing connection exists, should be able to setup a new connection and switch to it
- Should not get stuck on the connecting animation.

**Comment by jbennett88** [comment]
I have the same issue. I am not even connected to WiFi at all and it does that. I can't get my Mark 2 to connect to any WiFi, only way for me to connect it to a network is via LAN cable. Even at first boot, when I connect to the Mark2 hot spot, select my WiFi name and password, and then nothing happens. Doesn't even try to connect, the Mark2 hot spot stays on.

**Comment by ricmm** [comment]
Hey, sounds like two things:
1. @krisgesling seems to have managed to connect to a network at some point, but cannot "change" networks to a different one.
2. @jbennett88 seems to never have been able to use his Wifi at all.

@jbennett88 have you tried multiple different Wifi networks? different APs altogether preferable. Maybe create a hotspot with a phone and see if you can connect to that? -- just to guarantee you can indeed connect to something with that device.

**Comment by jbennett88** [comment]
@ricmm Yes, I have tried different APs altogether and it made no differences. PiCroft have no problem connecting to the same APs running on the same Pi board, so it's not the board. If I use the UI to try and connect to an AP, it does the same as what @krisgesling described, it just shows the animation and nothing happens, only way to get out of that is to restart the unit.
